README.md

```diff
@@ -24,26 +24,39 @@ git clone https://github.com/Thrillberg/imperial.git
 ```
 cd imperial
 ```
-3. Run the Rails setup command:
+3. Begin the PostgreSQL service:
+```
+sudo service postgresql start
```

Comment:
Does this command work? I would have thought it would be this (if pg is installed with homebrew):
```
brew services postgresql start
```
In any event, I'd like to avoid invocations of `sudo` here if possible.

Reply:
Oh that makes sense! If you want to specify that this line works where `apt install` is used rather than `brew`, that would be fine. I think `brew` is popular but obviously not the _only_ choice 😄 Out of curiosity, is the `sudo` necessary?

Reply:
Unfortunately `sudo` is necessary in the case where it's installed through `apt`

How does this look
[ef71461](https://github.com/Thrillberg/imperial/pull/682/commits/ef714611f3e1c8fdc8eec61b117aaee750dbad77)

Alternatively, Im happy to remove the `apt` lines too. I'd imagine some 90% of the ppl would use `brew`. I was just bumbling my way through and installing through `brew` never occurred to me

Reply:
Ok sounds good! Let's keep the `apt` lines in, just in case.

I made a mistake here! It should be `brew services start postgresql`. I was just going off memory before but I just confirmed that it works this way.

Reply:
Fixed in
[bd7aa08](https://github.com/Thrillberg/imperial/pull/682/commits/bd7aa08114e498b8cf0ba3a42e2d1e8011477293)